index.html

```diff
@@ -0,0 +1,1255 @@
+          <li class="px-3 my-2"><a href="#" class="text-decoration-none text-secondary">Coronavirus</a></li>
+          <li class="px-3 my-2"><a href="#" class="text-decoration-none text-secondary">Stimulus Package</a></li>
+          <li class="px-3 my-2"><a href="#" class="text-decoration-none text-secondary">Vaccine</a></li>
+          <li class="px-3 my-2"><a href="#" class="text-decoration-none text-secondary">Capitol Riots</a></li>
+        </ul>
+      </div>
+      </nav>
+    </header>
+    <main class="main-content container-fluid">
+      <div class="colwrap row px-2 px-lg-3">
+        <div class="col-left row">
+          <div class="col-left-1 col-sm-12 col-md-5 col-lg-5 pt-4">
+            <section class="featured-stories">
+              <h3 class="fs-6 fw-bold text-secondary">FEATURED STORIES</h3>
+
+              <div class="story mb-3">
+                <div class="stories-img d-block position-relative mb-1">
+                  <a href="#"><img src="resources/img/donald-trump.jpg" alt="Donald Trump" class="img-fluid story-picture position-relative d-block mb-2"></a>
+                  <div class="position-absolute bottom-0 start-0 bg-light px-3 py-1 text-danger fw-bold fs-6">U.S.</div>
+                </div>
+                <h3 class="story-title mb-2 fw-bold fs-5"><a href="#" class="text-decoration-none text-dark">The 73 People Pardoned by Donald Trump on His Last Day As President</a></h3>
+                <div class="story-description">
+                  Notable people pardoned by Trump today include former White House chief strategist Steve Bannon and rapper Lil Wayne.
+                </div>
+              </div>
+
+              <div class="story mb-3">
+                <div class="stories-img d-block position-relative mb-1">
+                  <a href="#"><img src="resources/img/donald-trump2.webp" alt="Donald Trump" class="img-fluid story-picture position-relative d-block mb-2"></a>
+                  <div class="position-absolute bottom-0 start-0 bg-light px-3 py-1 text-danger fw-bold fs-6">POLITICS</div>
+                </div>
+                <h3 class="story-title mb-2 fw-bold fs-5"><a href="#" class="text-decoration-none text-dark">Donald Trump Can Issue Secret Pardons, but They're Risky</a></h3>
+                <div class="story-description">
+                  Trump chose not to issue pardons for himself of his personal attorney, former New York Mayor Rudy Giuliani.
+                </div>
+              </div>
+
+              <div class="story mb-3">
+                <div class="stories-img d-block position-relative mb-1">
+                  <a href="#"><img src="resources/img/newsweekexpert.webp" alt="Newsweek" class="img-fluid story-picture position-relative d-block mb-2"></a>
+                  <div class="position-absolute bottom-0 start-0 bg-light px-3 py-1 text-danger fw-bold fs-6">EXPERTS</div>
+                </div>
```

Comment:
- [ ] I see that you are using the `text-danger` and `button-danger` a lot of times to get a red text and background in your clone, while using Bootstrap classes it's great it can be a problem in this case.
The shade of red the NewsWeek site uses is not the same red the Bootstrap classes have.

Please go through your clone and make sure that all the places where the shade of red differ from the original site are fixed.